docs/content/en/docs/getting-started/optional/gitops.md

```diff
@@ -147,9 +147,11 @@ spec:
 
 ### git Configuration Spec Details
 ### repositoryUrl (required)
->**_NOTE:_** The `repositoryUrl` value for private SSH repositories is of the format `ssh://git@provider.com/$REPO_OWNER/$REPO_NAME.git`. This may differ from the default SSH URL given by your provider. For example, the github.com user interface provides an SSH URL containing a `:` before the repository owner, rather than a `/`. Make sure to replace this `:` with a `/`, if present.
 * __Description__: The URL of an existing repository where EKS Anywhere will store your cluster configuration and sync it to the cluster. For private repositories, the SSH URL will be of the format `ssh://git@provider.com/$REPO_OWNER/$REPO_NAME.git`
 * __Type__: string
+* __Value__: The `repositoryUrl` value is of the format `ssh://git@provider.com/$REPO_OWNER/$REPO_NAME.git`. This may differ from the default SSH URL given by your provider. 
+  * The github.com user interface provides an SSH URL containing a `:` before the repository owner, rather than a `/`. Make sure to replace this `:` with a `/`, if present. 
+  * The CodeCommit SSH URL must include SSH-KEY-ID in format `ssh://<SSH-Key-ID>@git-codecommit.<region>.amazonaws.com/v1/repos/<repository>`.
```

Comment:
would it make sense for us to explicitly add a section to the docs for code commit vs github?

Reply:
What other info do you want to include in that section?

Reply:
I suggested a sentence to call out the two different URLs. If there isn't more you need to say, I don't think a separate section is needed.